packages/Adapters/Moab/test/tstMoabEntityLocalMap.cpp

```diff
@@ -340,6 +340,7 @@ TEUCHOS_UNIT_TEST( MoabEntity, quad_4_test )
 
     // Test the mapping to reference frame.
     Teuchos::Array<double> ref_good_point( space_dim );
+    // This lines throws. It seems that it frees an internal Teuchos::Array.
```

Comment:
Yes. This is just temporary until we decide what to with this line, fix it or no? I don't know the adapter for Moab, so I am not sure if it is worth spending time on this.